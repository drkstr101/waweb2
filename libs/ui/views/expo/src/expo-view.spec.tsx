import { render } from '@testing-library/react';

import ExpoView from './expo-view';

describe('ExpoView', () => {
  it('should render successfully', () => {
    const { container } = render(<ExpoView />);
    expect(container).toBeInstanceOf(HTMLElement);
    expect(container).toMatchSnapshot();
  });
});
