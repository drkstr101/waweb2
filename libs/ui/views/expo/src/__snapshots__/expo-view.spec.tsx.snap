// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ExpoView should render successfully 1`] = `
<div>
  <div
    class="container"
  >
    <h1>
      Welcome to ExpoView!
    </h1>
  </div>
</div>
`;
